evap/staff/tests/test_views.py

```diff
@@ -2394,6 +2394,13 @@ def test_evaluation_confirm_without_main_language(self):
 
         self.assertEqual(Evaluation.objects.first().state, self.evaluation.State.APPROVED)
 
+    def test_general_contribution_log_entry(self):
+        evaluation = baker.make(Evaluation)
+        page = self.app.get(reverse("staff:evaluation_edit", args=[evaluation.pk]), user=self.manager)
```

Comment:
In this test class, there is already an evaluation (and corresponding url) set up in `setUpTestData` which we can use here like so:
```suggestion
        page = self.app.get(self.url, user=self.manager)
```
Or did you create a separate evaluation on purpose and I am not understanding something?